docs/tutorials/Usage/domain_tutorial.jl

```diff
@@ -118,16 +118,18 @@ for example) will be [θ11, θ12; θ21, θ22]. Your variable always has the same
 ## Future work
 Almost all interactions between variables in land surface models are within column - that is, there is only
 vertical transport and exchanges. The exception to this is the horizontal flow of water on the surface
-and within the soil. The `right hand side` (rhs) functions (the ODE functions) can be split into
-"vertical" and "horizontal" pieces. We envision each step of the land surface model simulation to be solved  
-in two steps: (1) the vertical rhs evaluations are carried out (and can be parallelized), and (2) the horizontal 
-rhs functions are then evaluated (possibly less frequently?) and require communcation between columns.
-In this case, right hand side functions will need to be aware of the domain.
+and within the soil. The tendency (produced by `make_exp_tendency` and `make_imp_tendency`) functions
+(the ODE functions) can be split into "vertical" and "horizontal" pieces. We envision each step of the
```

Comment:
"We envision each step of the
land surface model simulation to be solved  in two steps: (1) the vertical tendency evaluations are carried out
(and can be parallelized), and (2) the horizontal tendency functions are then evaluated (possibly less frequently?)
and require communication between columns. 

In general, tendencies reflecting horizontal flow will be treated explicitly and include in the explicit tendency function. Tendencies reflecting vertical flow may be treated explicitly or implicitly depending on the use case. To solve the problem, we then use IMEX (mixed explicit/implicit) methods."